Sources/PackageLoading/TargetSourcesBuilder.swift

```diff
@@ -185,26 +189,31 @@ public struct TargetSourcesBuilder {
     }
 
     /// Compute the rule for the given path.
-    private func computeRule(for path: AbsolutePath) -> FileRuleDescription.Rule {
+    public static func computeRule(for path: AbsolutePath, toolsVersion: ToolsVersion, additionalFileRules: [FileRuleDescription]) -> FileRuleDescription.Rule {
```

Comment:
Is this the one that clients of libSwiftPM would now call for output files from plugins?

Reply:
I think so